cns/cnsclient/cli.go

```diff
@@ -0,0 +1,96 @@
+package cnsclient
+
+import (
+	"fmt"
+	"net"
+	"sort"
+
+	"github.com/Azure/azure-container-networking/cns"
+	acn "github.com/Azure/azure-container-networking/common"
+)
+
+var (
+	getArg          = "get"
+	getAvailableArg = "Available"
+	getAllocatedArg = "Allocated"
+	getAllArg       = "All"
+
+	releaseArg = "release"
+)
+
+func HandleCNSClientCommands(cmd, arg string) {
+	var ip net.IP
+
+	// retrieve the primary interface that CNS is listening on
+	interfaces, _ := net.Interfaces()
+	for _, iface := range interfaces {
+		addrs, _ := iface.Addrs()
+		for _, address := range addrs {
+			if ipnet, ok := address.(*net.IPNet); ok && !ipnet.IP.IsLoopback() && ipnet.IP.To4() != nil {
+				ip = ipnet.IP
+			}
+		}
+	}
+
+	cnsurl := "http://" + ip.String() + ":10090"
+	cnsClient, _ := InitCnsClient(cnsurl)
```

Comment:
is the second return value error?

should log it and exit 1 if an error happened